graylog2-web-interface/src/components/indexers/IndexerClusterHealth.tsx

```diff
@@ -84,14 +90,10 @@ const IndexerClusterHealth = ({ minimal }: Props) => {
     <Row className="content">
       <Col md={12}>
         {!minimal && (
-          <>
+          <Header>
             <h2>Elasticsearch cluster</h2>
-
-            <SmallSupportLink>
-              The possible Elasticsearch cluster states and more related information is available in the{' '}
-              <DocumentationLink page={DocsHelper.PAGES.CONFIGURING_ES} text="Graylog documentation" />.
-            </SmallSupportLink>
-          </>
+            <DocumentationLink page={DocsHelper.PAGES.CONFIGURING_ES} text="Indexer failures documentation" displayIcon />
```

Comment:
Shouldn't the `text` be: `Elasticsearch setup documentation`?

Reply:
Good point, I fixed it.